configure.ac

```diff
@@ -235,6 +236,12 @@ if test "x$with_di_betabmp" == xyes; then
    fi
 fi
 
+if test "x$with_di_betarislive" == xyes; then
+   if test x"$with_transport_kafka" != xyes; then
+      AC_MSG_ERROR([(Beta) BMP Stream data interface requires Kafka transport module (--without-betarislive to disable)])
```

Comment:
not sure what the deps should be for this DI, but this seems to be a copy/paste error

Reply:
the kafka requirement is removed in 711460b